src/lib.rs

```diff
@@ -43,6 +44,7 @@ pub extern "C" fn _PG_init() {
     };
 
     GUCS.init("pg_analytics");
+    pg_shmem_init!(env::DUCKDB_CONNECTION_CACHE);
 
     // TODO: Change to ParadeExtension::PgAnalytics
```

Comment:
I've just noticed this here. I've updated this in my PR in the monorepo: https://github.com/paradedb/paradedb/pull/1440. Is this going to fail if the `shared/` create isn't present?

Reply:
Nvm I opened a separate issue for this